pkgs/tools/misc/fiano/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
```

```suggestion
}:

buildGoModule rec {
```
Just a trivial point, but could you please add a blank line here?